.github/workflows/checks.yml

```diff
@@ -57,6 +57,24 @@ jobs:
           ignore_words_list: msdos, pullrequest
           skip: ./.git,coverity,rpmbuild,samples
 
+  source:
+    name: "Source code checks"
+    runs-on: ubuntu-latest
+    steps:
+    - name: "Clone Repository"
+      uses: actions/checkout@v2
+    - name: "Run Tests"
+      uses: osbuild/containers/src/actions/privdocker@e4de123f43b95e99dfe8eed0bd5a1cd58db50715
+      with:
+        image: ghcr.io/osbuild/osbuild-ci:latest-202208041541
+        run: |
+          git config --global --add safe.directory /osb/workdir
+          python3 -m pytest \
+            test/src/test_pylint.py \
+            -k autopep8 \
+            --rootdir=. \
+            -v
```

Comment:
Isn't it now run twice, also as part of `test.src` in `tests.yml`?

Reply:
No, because the old container does not have `autopep8` and thus it is skipped. Ideally I would move the pylint test over to `checks.yml` as well, but the new container has a newer pylint and we get a ton of new warnings (mostly missing `encoding` on `open` calls).

Reply:
Right.